src/test/java/com/synopsys/integration/blackduck/dockerinspector/IntegrationTestCommon.java

```diff
@@ -214,7 +214,7 @@ public static void testImage(final Random random, final ProgramVersion programVe
 
     private static List<String> createDockerInspectorCmd(final Random random, final ProgramVersion programVersion, final TestConfig.Mode mode, final String inspectTargetArg, final String repo, final String tag,
         final String codelocationName, final List<String> additionalArgs) throws IOException {
-        final List<String> cmd = new ArrayList<>();
+        List<String> cmd = new ArrayList<>();
```

Comment:
Purely for my own curiosity: Is the intent behind removing final to provide a hint to the reader that the contents of this array will be changed?